yanic/yanic.conf.tmpl

```diff
@@ -46,6 +53,14 @@ offline_after = "10m"
 enable   = true
 path = "/srv/yanic/meshviewer.json"
 
+[[nodes.output.meshviewer-ffrgb]]
```

Comment:
I think ffrgb should be replaced with ffdon here?

Reply:
This is the identifier of the output type I believe, basically "JSON for the FF RGB version of the meshviewer", so it should be correct like this. For the existing output it specifies the same name.